pykrige/ok.py

```diff
@@ -203,16 +203,30 @@ def __init__(self, x, y, z, variogram_model='linear', variogram_parameters=None,
         if self.enable_plotting and self.verbose:
             print("Plotting Enabled\n")
 
-        self.XCENTER = (np.amax(self.X_ORIG) + np.amin(self.X_ORIG))/2.0
-        self.YCENTER = (np.amax(self.Y_ORIG) + np.amin(self.Y_ORIG))/2.0
-        self.anisotropy_scaling = anisotropy_scaling
-        self.anisotropy_angle = anisotropy_angle
-        if self.verbose:
-            print("Adjusting data for anisotropy...")
-        self.X_ADJUSTED, self.Y_ADJUSTED = \
-            core.adjust_for_anisotropy(np.copy(self.X_ORIG), np.copy(self.Y_ORIG),
-                                       self.XCENTER, self.YCENTER,
-                                       self.anisotropy_scaling, self.anisotropy_angle)
+        self.coordinates_type = coordinates_type
+        if coordinates_type == 'euclidean':
+            self.XCENTER = (np.amax(self.X_ORIG) + np.amin(self.X_ORIG))/2.0
+            self.YCENTER = (np.amax(self.Y_ORIG) + np.amin(self.Y_ORIG))/2.0
+            self.anisotropy_scaling = anisotropy_scaling
+            self.anisotropy_angle = anisotropy_angle
+            if self.verbose:
+                print("Adjusting data for anisotropy...")
+            self.X_ADJUSTED, self.Y_ADJUSTED = \
+                core.adjust_for_anisotropy(np.copy(self.X_ORIG), np.copy(self.Y_ORIG),
+                                           self.XCENTER, self.YCENTER,
+                                           self.anisotropy_scaling, self.anisotropy_angle)
+        elif coordinates_type == 'geographic':
+            # Leave everything as is in geographic case.
+            # May be open to discussion?
+            self.XCENTER= 0.0
+            self.YCENTER= 0.0
+            self.anisotropy_scaling = 1.0
+            self.anisotropy_angle = 0.0

```

Comment:
Yes, I don't know how this should work. In any case we can't silently overwrite a user-provided value e.g. of `anisotropy_angle` without warning the user it's happening. If anisotropy is not compatible with this types of coordinate, the doc-string should say so.